Source/WebInspectorUI/UserInterface/Models/CSSGrouping.js

```diff
@@ -25,23 +25,48 @@
 
 WI.CSSGrouping = class CSSGrouping
 {
-    constructor(type, text, sourceCodeLocation)
+    constructor(nodeStyles, ownerStyleSheet, {id, type, text, sourceCodeLocation} = {})
     {
+        console.assert(nodeStyles);
         console.assert(Object.values(CSSGrouping.Type).includes(type));
         console.assert(!text || (typeof text === "string" && text.length));
         console.assert(!sourceCodeLocation || sourceCodeLocation instanceof WI.SourceCodeLocation);
 
+        this._nodeStyles = nodeStyles;
+        this._ownerStyleSheet = ownerStyleSheet || null;
```

Comment:
if this is not actually required then it should be in the `options = {}`

Reply:
ditto